src/main/scala/scorex/core/network/message/BasicMessagesRepo.scala

```diff
@@ -1,20 +1,21 @@
 package scorex.core.network.message
 
 
+import org.ergoplatform.modifiers.NetworkObjectTypeId
 import scorex.core.consensus.SyncInfo
 import scorex.core.network._
 import scorex.core.network.message.Message.MessageCode
 import scorex.core.serialization.ScorexSerializer
-import scorex.core.{ModifierTypeId, NodeViewModifier}
+import scorex.core.NodeViewModifier
 import scorex.util.Extensions._
 import scorex.util.serialization.{Reader, Writer}
 import scorex.util.{ModifierId, ScorexLogging, bytesToId, idToBytes}
 
 import scala.collection.immutable
 
-case class ModifiersData(typeId: ModifierTypeId, modifiers: Map[ModifierId, Array[Byte]])
+case class ModifiersData(typeId: NetworkObjectTypeId.Value, modifiers: Map[ModifierId, Array[Byte]])
```

Comment:
ScalaDoc missing, this and other classes

Reply:
Done (for this class touched only)